storage/innobase/xtrabackup/doc/source/.res/contents/example.xbstream.txt

```diff
@@ -5,12 +5,12 @@
    * - Task
      - Command
    * - Stream the backup into an archive named :file:`backup.xbstream`
-     - :bash:`xtrabackup --backup --stream=xbstream ./ > backup.tar`
+     - :bash:`xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`
    * - Stream the backup into a `compressed` archive named :file:`backup.xbstream`
-     - :bash:`xtrabackup --backup --stream=xbstream --compress ./ | gzip - > backup.tar.gz`
+     - :bash:`xtrabackup --backup --stream=xbstream --compress --target-dir=./ | gzip - > backup.xbstream`
```

Comment:
Also please remove gzip, because --compress already compressing the data:
xtrabackup --backup --stream=xbstream --compress --target-dir=./ > backup.xbstream

Reply:
Ready for review